monitoring/monitorlib/auth.py

```diff
@@ -95,7 +95,7 @@ def issue_token(self, intended_audience: str, scopes: List[str]) -> str:
             urllib.parse.quote(intended_audience),
             self._sub,
         )
-        response = self._oauth_session.post(url)
+        response = self._oauth_session.get(url)
```

Comment:
This was interesting. Earlier, a POST was done by _oauth_session, but because POST endpoint was not available GET endpoint was giving the response. When I made the POST endpoint, it started calling the POST endpoint. But, the request in this method is having the url for GET call. Hence, I switched to call get() method of _oauth_session so that hits the GET endpoint.